README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
 You can also [download the compiled and minified assets (CSS, JavaScript) from
 GitHub](docs/installation/installing-from-dist.md).
 
-## Browser support
+## Accessibility
 
-GOV.UK Frontend will allow you to build services that comply with the [guidance
-in the Service Manual][service-manual-browsers].
+The GOV.UK Design System team regularly tests GOV.UK Frontend with a range of users to make sure it’s as accessible as possible. You can read [the results of our most recent test for accessibility](https://design-system.service.gov.uk/accessibility/).
 
-Currently, GOV.UK Frontend officially supports the following browsers:
+However, your service will not automatically meet [level AA of WCAG 2.1](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag) if you use GOV.UK Frontend. You must still do research to check that your service meets accessibility requirements, especially if you extend or modify components.
 
-| Operating system | Browser                                | Support     |
-|----------------- |----------------------------------------|-------------|
-| Windows          | Internet Explorer 8-10                 | functional  |
-| Windows          | Internet Explorer 11                   | compliant   |
-| Windows          | Edge (latest 2 versions)               | compliant   |
-| Windows          | Google Chrome (latest 2 versions)      | compliant   |
-| Windows          | Mozilla Firefox (latest 2 versions)    | compliant   |
-| macOS            | Safari 9+                              | compliant   |
-| macOS            | Google Chrome (latest 2 versions)      | compliant   |
-| macOS            | Mozilla Firefox (latest 2 versions)    | compliant   |
-| iOS              | Safari for iOS 9.3+                    | compliant   |
-| iOS              | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Samsung Internet (latest 2 versions)   | compliant   |
+You should also use:
 
-‘Compliant’ means that the components must look as good and function as well as
-they do in other modern browsers.
+- [the JavaScript from GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
```

Comment:
At the moment the reason we call out Javascript in this way but not CSS isn't clear. Maybe we could say that in many cases the JS contains "hidden" accessibility fixes eg. the button is like this, even though it might not seen like a component that comes with JS.

The component Javascript also needs to be initialised. Maybe that's too much detail but I suspect that step is often missed out.

Reply:
The initialisation part is covered in the content in the link (https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript) - we could look at improving that content separately if it's currently being missed by users?

Could we also add content about the 'hidden' accessibility fixes on that page too? I'd like to keep all the usage/installation instructions in that page, rather than get into too much detail here.

Reply:
Okay sounds good 👍